en/docs/Learn/APISecurity/OAuth2/AOuth2Scopes/scope-whitelisting.md

```diff
@@ -2,53 +2,21 @@
 
-If you do not want a role validation for a scope in an API's request, add the scope under the `<OAuthConfigurations>` element in the `<APIM_HOME>/repository/conf/api-manager.xml` file and restart the server. It will be whitelisted. For example,
+If you do not want a role validation for a scope in an API's request, add the scope as `white_listed_scopes` in the `<APIM_HOME>/repository/conf/deployment.toml` file and restart the server. It will be whitelisted. For example,
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td><div class="container" title="Hint: double-click to select code">
-<div class="line number1 index0 alt2">
-<code class="java plain">                 &lt;ScopeWhitelist&gt;                </code>
-</div>
-<div class="line number2 index1 alt1">
-<code class="java spaces">                </code> <code class="java plain">                 &lt;Scope&gt;^device_.*&lt;/Scope&gt;                </code>
-</div>
-<div class="line number3 index2 alt2">
-<code class="java spaces">                </code> <code class="java plain">                 &lt;Scope&gt;some_random_scope&lt;/Scope&gt;                </code>
-</div>
-<div class="line number4 index3 alt1">
-<code class="java plain">                 &lt;/ScopeWhitelist&gt;                </code>
-</div>
-</div></td>
-</tr>
-</tbody>
-</table>
+```
+[apim.oauth_config]
+white_listed_scopes = ["^device_.*", "openid", "some_random_scope"]
+```
 
 Next, invoke the Token API to get a token for the scope that you just whitelisted. For example,
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td><div class="container" title="Hint: double-click to select code">
-<div class="line number1 index0 alt2">
-<code class="java plain">                 curl -k -d                </code> <code class="java string">                 &quot;grant_type=password&amp;username=admin&amp;password=admin&amp;scope=some_random_scope&quot;                </code> <code class="java plain">                 -H                </code> <code class="java string">                 &quot;Authorization: Basic WmRFUFBvZmZwYVFnR25ScG5iZldtcUtSS3IwYTpSaG5ocEVJYUVCMEN3T1FReWpiZTJwaDBzc1Vh&quot;                </code> <code class="java plain">                 -H                </code> <code class="java string">                 &quot;Content-Type: application/x-www-form-urlencoded&quot;                </code> <code class="java plain">                 https:                </code> <code class="java comments">                 //10.100.0.3:8243/token                </code>
-</div>
-</div></td>
-</tr>
-</tbody>
-</table>
+```
+curl -k -d  "grant_type=password&username=admin&password=admin&scope=some_random_scope" -H "Authorization: Basic WmRFUFBvZmZwYVFnR25ScG5iZldtcUtSS3IwYTpSaG5ocEVJYUVCMEN3T1FReWpiZTJwaDBzc1Vh" -H "Content-Type: application/x-www-form-urlencoded" https://10.100.0.3:8243/token
```

Comment:
I think it's good replace the hostname with a sample value without putting it like this.

Reply:
Fixed